serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -0,0 +1,225 @@
+= Data Index standalone
+:compat-mode!:
+// Metadata:
+:description: Data Index Service use as an independent service that allow to index and query audit data in {product_name}
+:keywords:  workflow, serverless, data, dataindex, data-index, index, service, standalone
+// External pages
+:distributions_url: https://quay.io/organization/kiegroup
+:dev_services_url: https://quarkus.io/guides/dev-services
+:test_containers_url: https://www.testcontainers.org/
+// Referenced documentation pages
+//Common constants
+:data_index_ref: Data Index
+:workflow_instance: process instance
+:workflow_instances: {workflow_instance}s
+
+[#data-index-service]
+== {data_index_ref} service deployment.
+
+{data_index_ref} service can be deployed referencing directly a distributed {data_index_ref} image. There are different images provided that take into account what persistence layer is required in each case.
+In each distribution, there are some properties to configure things like the connection with the database or the communication with other services.
+The goal is to configure the container to allow to process ProcessInstances and Jobs *events* that incorporate their related data, to index and store that in the database and finally, to provide the xref:data-index/data-index-core-concepts.adoc#data-index-graphql[{data_index_ref} GraphQL] endpoint to consume it.
+
+[#data-index-service-distributions]
+=== {data_index_ref} distributions
+{data_index_ref} service can be deployed referencing directly a distributed {data_index_ref} image. Here there are the different {data_index_ref} image distributions that can be found in link:{distributions_url}[Quay.io/kiegroup]:
+
+image::data-index/data-index-distributions.png[Image of data-index different available distributions]
+
+=== {data_index_ref} service deployment
+
+When the {data_index_ref} service is deployed, a `configMap` resource is created for the `application.properties` configuration of the {data_index_ref} service.
```

Comment:
Where is that ? How do you deploy it ?

Reply:
I've added some more details on the deployment process. (This part was like that in kie-docs)

Reply:
My point is: you are talking about a `configMap` and a service depoyed, but how to I deploy it and where is that configMap ?

Reply:
@radtriste maybe we can simplify removing any reference to that config map. As far as I know this was a way to comment how works the configuration related with the service. WDYT?

Reply:
Other option is to  add the link to this https://kubernetes.io/docs/concepts/configuration/configmap/ can clarify the idea of  that configMap, and link timeouts example with all the instructions in minikube.

Reply:
My main worry is that you never mention Kubernetes and then you write `When the {data_index_ref} service is deployed`. But where is it deployed ?
You example just above is docker-compose. There is no Kube deployment, nor any configmap yaml example

Reply:
+1 on @radtriste comment. @nmirasch it would be useful to explore the k8s deployment in another guide about data index. So there we can deep dive into this process and how to deploy it in the cloud, scale, configure persistence, etc. It will also help us identify the bottlenecks and requirements of deploying DI in the cloud so we can tailor the operator in the future to add support to it.

Reply:
@radtriste @ricardozanini  for now, I have added more details and descriptions on the commented standalone deployments resources and linked other guides and the timeout example that explain step by step how to deploy and use it. Please, Could you take a look and see if the information is properly sorted and meaningful?  
@ricardozanini  "it would be useful to explore the k8s deployment in another guide", I'm a bit lose with the pages structure right now. Do we need one more for configuration, for  Knative deployment, for Kafka deployment?  
Thanks both in advanced!

Reply:
That is better. Gonna resolve that conversation. We can continue on the new opened ones :)